src/themes/website/styles/widgets/textblock.scss

```diff
@@ -25,3 +25,10 @@ p:empty {
         position: relative;
     }
 }
+
+.description-text {
```

Comment:
A few comments here:
- There is `.not-configured` class already (it's used i.e. in Menu widget for similar purpose).
- `textblock.scss` contains "Text block" widget specific styles, this one looks more like for `utils.scss` file.
- if you assign a `background-color`, you may need to assign `color` as well, otherwise the block will inherit it from global stylesheet where customers can assign text colors that will be poorly visible on this background.